crane-cloud/backend
# feat(CI/CD): add docker webhook endpoint listener

# Description
Adds endpoint `/apps/docker_listener` to receive docker webhook notifications and update the user application images

## Trello Ticket ID
ticket link [here](https://trello.com/c/RjYWE6eu/1721-create-a-docker-webhook-listener-to-auto-deploy-applications)

## How Can This Be Tested?
Create a webhook in a docker repository, point it to the endpoint `/apps/docker_listener` then see how application will be updated automatically
